test/e2e-helm/e2e_test.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 The Operator-SDK Authors
```

Comment:
These filenames seem funky to me:
1. This file should be named `e2e_suite_test.go` since it is the global setup for the entire package, and all test files should end in `_test.go`
2. `e2e_suite.go` should be renamed to `e2e_test.go` since it is the file that contains the actual test logic.

And I _think_ we should be able to put these in the `_test` package, which has [special meaning for the Go compiler](https://github.com/golang/go/issues/25223).

Reply:
I really dislike the approach of we have these 2 files. My suggestion in order to follow the Go standard and keep it eaiser to be tested locally is we have : 

 | test/
├── e2e_helm_suite_test.go
├── e2e_ansible_suite_test.go
├── e2e_golang_suite_test.go

Etc ...

Reply:
The two-file approach is standard for ginkgo. We should follow this, and fill our tests in: https://github.com/onsi/ginkgo#bootstrapping

Reply:
The idea is:
1. package level file that sets up the suite
2. individual test files for each thing under test.

So in our case, I think we should have:
1. A suite file that creates the test context and initializes a new project in a `BeforeSuite`
2. Test files for each of the things we want to test.
 - Run the project
 - Generate the bundle
 - Run scorecard
 - etc.

Reply:
Make much more sense. I read the docs as well. Tks for pointing it out. Done 👍